### PR TITLE
Improve deployement part

### PR DESCRIPTION
**Pull request by Ben-Rey**

### CODE REVIEW

**Review by AurelienFT — COMMENTED** [review]
Can you also add the usage of is deploying contract in the contract example?

File: docs/build/quickstart.mdx [hunk 1]
```diff
@@ -57,7 +57,7 @@ In the meantime, here are the key components you should be familiar with:
 - **`.env.example`**: This file serves as a template for the `.env` file you'll need to create. It outlines the necessary environment variables required to deploy your smart contract.
 
 - **`package.json`**: This file contains the npm commands crucial for various operations such as checking, testing, building, and deploying your smart contract.
-:::
+  :::
```
Comment (julienbrs) [inline docs/build/quickstart.mdx:60]:
```suggestion
:::
```

File: docs/build/quickstart.mdx [hunk 2]
```diff
@@ -72,7 +72,7 @@ JSON_RPC_URL_PUBLIC=https://buildnet.massa.net/api/v2:33035
 
 - WALLET_SECRET_KEY: Store your wallet's private key used to deploy the contract here. Make sure your wallet is funded.
 - JSON_RPC_URL_PUBLIC: This is the URL of the JSON-RPC interface of a public or a private Massa node.
-:::
+  :::
```
Comment (julienbrs) [inline docs/build/quickstart.mdx:75]:
```suggestion
:::
```

**Review by julienbrs — APPROVED** [review]

**Review by AurelienFT — APPROVED** [review]

File: docs/build/quickstart.mdx [hunk 3]
```diff
@@ -126,6 +126,25 @@ npm run deploy
 
 Upon successful execution, the console should display the operation id that you can use to track the deployment of your smart contract.
 
+:::info
+After deployment, the event will be displayed in the console. This event contains the address of your deployed smart contract and the event message set in the constructor function.
```
Comment (qdrn) [inline docs/build/quickstart.mdx:130]: Which event ? It is never mentioned before so it feels a bit weird. I suggest making it a bit more clear in the `Writing and Deploying the Smart Contract` section. Something simple.

**Review by qdrn — CHANGES_REQUESTED** [review]

File: docs/build/quickstart.mdx (hunk 3 above)
Comment (Ben-Rey) [reply docs/build/quickstart.mdx]: I just changed "the" to "an". I try to add it in  Writing and Deploying the Smart Contract but it was repetitive.

**Review by qdrn — APPROVED** [review]

**Pull request merged**